package.json

```diff
@@ -10,9 +10,11 @@
   },
   "main": "index.js",
   "scripts": {
+    "build": "npm run clean:css",
     "example:fastatic": "examples/fastatic/index.js",
     "commit": "git-cz",
     "test": "tape test/*.js | tap-spec",
+    "clean:css": "cleancss -o src/assets/voorhoede-core.min.css src/assets/voorhoede-core.css",
```

Comment:
We normally don't put auto-generated code under version control. So I would output to the `dist/` directory. If you want a more advanced CSS build step (including autoprefixing and variables etc) you could consider our [PostCSS recipe](https://github.com/voorhoede/front-end-tooling-recipes/tree/master/postcss-process-and-watch).